src/test/java/teammates/test/automated/FeedbackSessionOpeningReminderTest.java

```diff
@@ -99,7 +99,8 @@ public void testAdditionOfTaskToTaskQueue() throws Exception {
         
         // Modify session to set emails as unsent but still open
         // by closing and opening the session. Also disable sending
-        // open emails.
+        // open emails, but in the current implementation open emails
```

Comment:
Agree, it can be confusing if the implementation is modified but the comment is not. But not including it will make people wonder when they found out that the open emails are still sent, like me yesterday. So it's a trade-off.